src/core/credentials/transport/tls/spiffe_utils.cc

```diff
@@ -156,4 +189,101 @@ absl::StatusOr<SpiffeId> SpiffeId::FromString(absl::string_view input) {
   return SpiffeId(trust_domain, absl::StrCat("/", path));
 }
 
+void SpiffeBundleKey::JsonPostLoad(const Json& json, const JsonArgs&,
+                                   ValidationErrors* errors) {
+  {
+    ValidationErrors::ScopedField field(errors, ".use");
+    if (use_ != kAllowedUse) {
+      errors->AddError(
+          absl::StrFormat("got %s. Only supported value for use field is %s.",
+                          use_, kAllowedUse));
+      return;
+    }
+  }
+  {
+    ValidationErrors::ScopedField field(errors, ".kty");
+    if (kty_ != kAllowedKty) {
+      errors->AddError(
+          absl::StrFormat("got %s. Only supported value for kty field is %s",
+                          kty_, kAllowedKty));
+      return;
+    }
+  }
+  {
+    ValidationErrors::ScopedField field(errors, ".x5c");
+    if (x5c_.size() != kx5cSize) {
+      errors->AddError(absl::StrFormat(
+          "got vector length %i. Expected length of exactly %i.", x5c_.size(),
+          kx5cSize));
+      return;
+    }
+    // absl::Status status = ValidateCertificateIsValidX509(x5c_[0]);
+
+    std::string pem_cert =
+        absl::StrCat(kCertificatePrefix, x5c_[0].data(), kCertificateSuffix);
+    auto certs = ParsePemCertificateChain(pem_cert);
+    if (!certs.ok()) {
+      errors->AddError(certs.status().message());
```

Comment:
We're dropping the status code here, should we stringify the entire absl::Status, e.g. using the ToString(...) method?

Reply:
Done